roles/systemd_exporter/meta/main.yml

```diff
@@ -0,0 +1,32 @@
+---
+galaxy_info:
+  author: Prometheus Community
+  description: Prometheus Systemd Exporter
+  license: Apache
+  company: none
+  min_ansible_version: "2.9"
+  platforms:
+    - name: Ubuntu
+      versions:
+        - bionic
+        - xenial
+    - name: Debian
+      versions:
+        - stretch
+        - buster
+    - name: EL
+      versions:
+        - '7'
+        - '8'
+    - name: Fedora
+      versions:
+        - '30'
+        - '31'
```

Comment:
Non-blocking note: We should probably sync these lists with the platforms we're currently testing with.

Reply:
Yep, planning to update this for all the roles in #53